tests/test_executable.py

```diff
@@ -129,6 +129,7 @@ def test_executable_not_privileged(
     exit_code, output = application_docker_container.exec_run(cmd=command, privileged=False, user="root:root")
 
     _no_errors(output.decode())
+    print(output.decode())
```

Comment:
Can swap this line with the previous (so we get the output print, before calling `_no_errors`)